multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/util/ApplicationColorDetectorTest.java

```diff
@@ -135,9 +131,9 @@ void detectSingularDeployedApp3ModulesNoSuffix() {
     @Test
     void detectSingularDeployedApp3ModulesGreenBlueSuffix() {
         DeployedMta deployedMta = createMta("com.sap.sample.mta.consumer", Collections.emptySet(),
-                                            List.of(createMtaApplication("app-1", "app-1-blue", parseDate("2016-15-10")),
+                                            List.of(createMtaApplication("app-1", "app-1-blue", parseDate("2016-10-10")),
```

Comment:
Is there a reason why month was changed?

Reply:
yes 15 is invalid month and the method (in the test) was just printing stack traces for no reason and returning nulls